src/lab3/ReaderOBJ.ts

```diff
@@ -0,0 +1,50 @@
+import fs, { ReadStream } from 'fs';
+import Mesh from './structures/mesh/Mesh';
+import readline from 'readline';
+import Vertex3D from '../lab1/structures/vertex/Vertex3D';
+import Triangle from './structures/triangle/Triangle';
+
+export default class ReaderOBJ {
+  public static async read(pathToFile: string): Promise<Mesh> {
+    const stream = fs.createReadStream(pathToFile);
+    return ReaderOBJ.readStream(stream);
+  }
```

Comment:
I think there might be complaints as to why this class deals with file system from George